.github/workflows/build.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - update-optout-fields
```

Comment:
I assume this was temporarily needed?

Reply:
Yea, I wanted to push my branch to QA so I can test, without affecting the main branch while people were busy trying to get V3 live, but I think that ship has sailed now